helm/hpcc/templates/eclccserver.yaml

```diff
@@ -30,6 +30,7 @@ spec:
                 {{ include "hpcc.daliArg" $ }}
               ]
 {{- include "hpcc.addSecurityContext" (dict "root" $ "me" .) | indent 8 }}
+{{- include "hpcc.addResources" (dict "root" $ "me" .) | indent 8 }}
```

Comment:
This is a bit confusing.  If useChildProcesses is defined the memory for eclccserver is defined one way, if it is false it is defined a different way.  It would be easier to configure and understand if a single setting to apply to either.

Reply:
Eclccserver's memory is always defined by "resources".  The "compile" pod's resources are defined by ChildContainerResources.  How would you suggest it works?